frontend/src/views/account-mgmt/FacilityView.vue

```diff
@@ -0,0 +1,271 @@
+        </v-row>
+      </v-col>
+      <v-col cols="2">
+        <v-row>
+          <AppButton id="cancel" :primary="true" size="large" :to="{ name: 'organization-facility' }" :loading="loading" class="mr-6">Cancel</AppButton>
+          <AppButton id="save" :disabled="true" :primary="true" size="large" :to="{ name: 'account-mgmt' }" :loading="loading">Save</AppButton>
+        </v-row>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import AppButton from '@/components/ui/AppButton.vue'
+import AppLabel from '@/components/ui/AppLabel.vue'
+import FacilityInfo from '@/components/facilities/FacilityInfo.vue'
+import FacilityService from '@/services/facilityService'
+import alertMixin from '@/mixins/alertMixin'
+import { useAppStore } from '@/stores/app'
+import { mapState } from 'pinia'
+import { useAuthStore } from '@/stores/auth'
+
+export default {
+  name: 'FacilityView',
+  components: { AppButton, AppLabel, FacilityInfo },
+  mixins: [alertMixin],
+  data() {
+    return {
+      facilityId: null,
+      licences: undefined,
+      contacts: undefined,
+      loading: false,
+    }
+  },
+  computed: {
+    ...mapState(useAuthStore, ['userInfo']),
+    ...mapState(useAppStore, ['healthAuthorities']),
+    expenseAuthorities() {
+      return this.contacts?.filter(contact => contact.isExpenseAuthority)
+    },
+    primaryContact() {
+      return this.contacts?.filter(contact => contact.isPrimaryContact)
+    },
+    isEditablePrimaryContact() {
+      return this.primaryContact && this.primaryContact.length !== 0
+    },
+    isEditableExpenseAuthority() {
+      return this.expenseAuthorities && this.expenseAuthorities.length !== 0
+    },
+  },
+  created() {
+    this.facilityId = this.$route.params.facilityId
+    this.loadData()
+  },
+  methods: {
+    /**
+     * Load the data for the page
+     */
+    async loadData() {
+      try {
+        this.loading = true
+        this.getContacts()
+        this.getLicences()
+      } finally {
+        this.loading = false
+      }
+    },
+
+    /**
+     * Get the contacts (Expense Authorities and Primary Contact) for the facility
+     */
+    async getContacts() {
+      try {
+        this.contacts = await FacilityService.getContacts(this.facilityId)
+      } catch (error) {
+        this.setFailureAlert('Failed to get Primary and Expense Authority contacts for facilityId = ' + this.facilityId, error)
+      }
+    },
+
+    /**
+     * Get the licences for the facility
+     */
+    async getLicences() {
+      try {
+        this.licences = await FacilityService.getLicences(this.facilityId)
+      } catch (error) {
+        this.setFailureAlert('Failed to licence(s) for facilityId = ' + this.facilityId, error)
+      }
+    },
+
+
+    /**
+     * Get the role name by id
+     */
+    getRoleNameById(roleId) {
```

Comment:
You should be able to access this directly via mapGetters. It looks like this ManageUsersView.vue might do this as well.

Reply:
I'm not sure how to access the getter other than appStore.getRoleNameById(Number(roleId))?

Reply:
Based on discussions will leave this for now.